website/src/styles/themes.css

```
@@ -0,0 +1,242 @@
+ *
+ * The `.rc-scope` block mirrors renCal's theme system: the ren defaults from
+ * the app's src/global.css `:root` block, plus the derived-token chain from
+ * its `body` block. Theme primitives are applied as inline styles on top of
+ * this scope (the website equivalent of the app's `[data-theme]` wrapping),
+ * so the derivations react to them exactly like in the app.
+ * Keep the values in sync with src/global.css.
+ */
+
+.rc-scope {
+  --mono: "Geist Mono", ui-monospace, monospace;
+  --sans: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)